### PR TITLE
modernise configure.ac a bit

### PR DESCRIPTION
**Pull request by flammie**
after I used apertium-init today autotools scolded me of these outdated autoconf stuff, this is pretty much what autoupdate did. I also changed bug-report from email to github issues cause that's what people use these days.

### CODE REVIEW

**Comment by mr-martian** [comment]
Checking this, I just noticed that the monolinguals all have a `.pc.in` but bilingual doesn't - should bilingual have that as well?

**Comment by flammie** [comment]
> Checking this, I just noticed that the monolinguals all have a `.pc.in` but bilingual doesn't - should bilingual have that as well?

it would make sense, I guess since we have bilinguals actually depending on monolinguals but nothing depends on bilingual modules yet so it hasn't been strictly necessary but would be useful

**Comment by TinoDidriksen** [comment]
> Should bilingual have `.pc.in` as well?

Only if something else build-depends on them, which so far has not been the case. And I can't off-hand think of a case where it would happen.

File: bilingual-module/configure.ac [hunk 1]
```diff
@@ -1,6 +1,6 @@
 AC_PREREQ(2.52)
 
-AC_INIT([Apertium {{languageName1}}-{{languageName2}}], [0.1.0], [{{email}}], [apertium-{{languageCode1}}-{{languageCode2}}], [https://wiki.apertium.org/wiki/Apertium-{{languageCode1}}-{{languageCode2}}])
+AC_INIT([Apertium {{languageName1}}-{{languageName2}}],[0.1.0],[https://github.com/apertium/apertium-{{languageCode1}}-{{languageCode2}}/issues], [apertium-{{languageCode1}}-{{languageCode2}}], [https://wiki.apertium.org/wiki/Apertium-{{languageCode1}}-{{languageCode2}}])
```
Comment (TinoDidriksen) [inline bilingual-module/configure.ac:3]: +spaces

**Review by TinoDidriksen — APPROVED** [review]

**Pull request merged**